packages/frontend/src/components/organisms/MemberForm.tsx

```diff
@@ -194,21 +194,13 @@ export const MemberForm = (props: IProps) => {
               name="memberLabels"
               className="form-control"
             >
-              {availableMemberLabels.map((label) => (
+              {[...availableMemberLabels].map((label) => (
```

Comment:
Why do you use spread operator here?

Reply:
it's an old remnant of bug I was trying correct, I will remove it